### ISSUE TITLE
Seperate pages for unlinked tokens

### ISSUE THREAD

**Issue by spyrella** [League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima#161]
**User Story:**
Create new pages in journal entries for unlinked tokens of the same actor name.

**User Examples:**
Make it so there are separate pages for each different unlinked actor for unlinked actors. We'll use token id to differentiate between them.

**Comment by Vesselchuck** [comment]
Would that make a new journal entry for every Gray Howler? For example, if there would be 4 unlinked Gray Howlers with numeration, what would be the outcome?

**Comment by spyrella** [comment]
No, all npcs that share the same name would be under one journal entry, a new page within the journal entry will be created.

**Comment by Frozedon** [comment]
The implementations of the Adversaries and Codex sections of the Party sheet have made this feature obsolete.
